ENIGMAsystem/SHELL/Platforms/Win32/WINDOWScallback.cpp

```diff
@@ -36,10 +35,6 @@ using std::map;
 #endif
 
 namespace enigma_user {
-extern int keyboard_key;
-extern int keyboard_lastkey;
-extern string keyboard_lastchar;
-extern string keyboard_string;
```

Comment:
For some reason these are split between `PFmain.h` and `PFwindow.h` but regardless, we don't need the extra trash linger around.